src/Storages/ObjectStorage/StorageObjectStorage.h

```diff
@@ -276,6 +299,12 @@ class StorageObjectStorage::Configuration
     String format = "auto";
     String compression_method = "auto";
     String structure = "auto";
+    std::string partition_strategy_name = "wildcard";
+    /*
+     * Only supported by hive partitioning style for now
+     */
```

Comment:
```suggestion
     /// Only supported by hive partitioning style for now
```
For single line comments we use `///`